<launch>
    <group ns="picar">
        <group ns="camera_settings">
            <param name="width" value="80"/>
            <param name="height" value="60"/>
            <param name="framerate" value="90"/>
            <param name="racemode" value="1"/>
        </group>

        <node pkg="raspi_nodes" name="camera_node" type="camera_node.py" output="screen"/>
        <node pkg="raspi_nodes" name="motor_control" type="motor_control.py" output="screen"/>
    </group>

</launch>